src/streaming/models/CmcdModel.js

```diff
@@ -497,7 +520,8 @@ function CmcdModel() {
         let cid = settings.get().streaming.cmcd.cid ? settings.get().streaming.cmcd.cid : internalData.cid;
         cid = cmcdParametersFromManifest.contentID ? cmcdParametersFromManifest.contentID : cid;
 
-        data.v = CMCD_VERSION;
+        let v = settings.get().streaming.cmcd.reporting.requestMode.version ?? DEFAULT_CMCD_VERSION;
```

Comment:
Why we are using `let` instead `const` here? 
Also, why we defined another variable instead of assigning it directly to data.v?